tests/thread-safety/source.ml

```diff
@@ -0,0 +1,13 @@
+let () =
+  let rec repeat n =
+    if n <= 0 then ()
+    else repeat (n - 1)
+  in
```

Comment:
AFAIR OCaml threads can be preempted only if there is allocation.

And from experience, to make sure threads preempt each other doing some I/O would be best.

(maybe the code added by `bisect_ppx` is enough actually)